### DISCUSSION TITLE
The VSCode extension context is always located in the directory where the uno.config.ts is located, is there any way I can specify it myself?

### DISCUSSION THREAD

**Discussion by mutuguangda** [unocss/unocss#4343] [Q&A]
For example, if I create a uno.config.ts in the `config` directory, the VSCode extension can't automatically prompt the files in my `src` directory.
I know that the community prefers to create uno.config.ts in the root directory, but for some reasons I want to know whether this method is supported. Looking forward to the reply.

**Comment by ntnyq** [comment]
Currently, it seems that you can only put a config file in root and re-export or point the real config file.

See https://github.com/unocss/unocss/issues/2603.

**Reply by mutuguangda** [reply]
Thanks for the reply, so is what I'm doing so now
